src/app/shared/admin/admin-candidates/admin-elections-candidate-modal.component.html

```diff
@@ -17,9 +17,9 @@ <h3>Candidates</h3>
         </tr>
       </thead>
       <tbody *ngIf="candidates" >
-        <tr admin-candidates-row *ngFor="let row of candidates" [rowData]="row" [positions]="positions" [electionID]="electionID"  [election_type]="election_type" (remove)="removeCandidate($event)" style="overflow: scroll"></tr>
+        <tr admin-candidates-row *ngFor="let row of candidates" [rowData]="row" [positions]="positions" [electionID]="electionID"  [election_type]="election_type" (remove)="removeCandidate($event)" (notSaved)="enableNewCandidate($event)" style="overflow: scroll"></tr>
```

Comment:
Does this work without the $event argument? The method definition in the ts file doesn't have any parameters.

Reply:
yes it is, I updated the function with a doc string that includes the parameter.